subgraph/package.json

```diff
@@ -13,14 +13,15 @@
     "lint": "eslint --resolve-plugins-relative-to ../packages/eslint-config --ext .ts,.js src/ bin/"
   },
   "dependencies": {
-    "@graphprotocol/graph-cli": "0.22.2",
-    "@graphprotocol/graph-ts": "0.22.1",
+    "@graphprotocol/graph-cli": "0.21.1",
```

Comment:
Downgrading because new version breaks :/ (we do not have good tests to check this in CI)